tools/install_deepgemm.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# Script to install DeepGEMM from source
+# This script can be used both in Docker builds and by users locally
+
+set -e
+
+# Default values
+DEEPGEMM_GIT_REPO="https://github.com/deepseek-ai/DeepGEMM.git"
+DEEPGEMM_GIT_REF="7b6b5563b9d4c1ae07ffbce7f78ad3ac9204827c"
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --repo)
+            DEEPGEMM_GIT_REPO="$2"
+            shift 2
+            ;;
```

Comment:
I am thinking since we have the file_name install deepgemm, then we don't need to have a `--repo` here.